extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -97,10 +97,25 @@
             "outputs": [],
             "execution_count": null
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## 3. Format toc.yml file of the book\r\n",
+                "\r\n",
+                "Create a valid toc.yml file that enables your book to be opened on our view-let.\r\n",
+                "\r\n",
+                "<span style=\"color:red\">Note:</span> Skipping this step would have ill-formed toc and need your manual updates to it."
+            ],
+            "metadata": {
+                "azdata_cell_guid": "5a29e2dd-d9f5-4dc1-87bc-bce639362f51"
+            }
+        },
         {
             "cell_type": "code",
             "source": [
                 "# Update toc file, book title and clean up the directores\n",
+                "\n",
+                "from os import path\n",
```

Comment:
You have a mix of os.path and just path. I'd suggest just importing path like this up above and then converting all the os.path in the notebook to just use path since that's a lot cleaner (but do that in a separate PR - maybe jus tmake one with the negative indexing/other cleanup too)